pkg/serverless/metrics/statfs_windows.go

```diff
@@ -0,0 +1,12 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
+
+package metrics
+
+func statfs(_ string) (float64, float64, float64, error) {
+	return 0, 0, 0, nil
```

Comment:
Nit.

Do we even really need this with the windows build tag? We never build the extension using the windows build tag, so this code will never be used.

Reply:
Unfortunately, yeah we need it. Otherwise windows linters and tests will fail (e.g. [here](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/561624958)) if there's no windows stub (and will also fail on the unix-specific package if found)

Reply:
ahha, totally makes sense